TACC-Cloud/hazmapper
# task/WG-288: improve configuration of DesignSafe url

## Overview: ##

This PR:
* improve configuration of DesignSafe URL (WG-288)
* fixes links to DS (WG-322)

## PR Status: ##



## Related Jira tickets: ##

* [WG-288](https://tacc-main.atlassian.net/browse/WG-288)
* [WG-322](https://tacc-main.atlassian.net/browse/WG-322)

## Testing Steps: ##
1. Review code for WG-288 and also see that still works with DesignSafe next locally (and when deployed to /exp)
2. For WG-322, see the links from maps connected to My Data (and maps connected to projects) work and open in new tab (see https://tacc-main.atlassian.net/browse/WG-322 screenshot of where this is in the UI).
